ddm_test_validate.py

```diff
@@ -0,0 +1,214 @@
+import logging
+import datetime
+import pathlib
+import os
+import time
+from distutils.util import strtobool
+from typing import Any, Dict, List
+
+import numpy as np
+import pandas as pd
+from azure.core.exceptions import HttpResponseError
+from dotenv import load_dotenv, set_key
+from microsoft_bonsai_api.simulator.client import BonsaiClient, BonsaiClientConfig
+from microsoft_bonsai_api.simulator.generated.models import (
+    SimulatorInterface,
+    SimulatorSessionResponse,
+    SimulatorState,
+)
+
+from base import BaseModel
+
+logging.basicConfig()
+logging.root.setLevel(logging.INFO)
+logger = logging.getLogger("datamodeler")
+
+import hydra
+from omegaconf import DictConfig
+
+from model_loader import available_models
+
+from sim.moab.moab_main import SimulatorSession,  env_setup
+from sim.moab.policies import coast, random_policy
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+env_name = "DDM"
+log_path = "logs"
+
+
+class Simulator(BaseModel):
+    def __init__(self, model, states=List[str], actions=List[str], configs=List[str], 
+        log_file: str = None,):
+        self.dd_model = model
+        self.features = states + actions + configs
+        self.labels = states
+        self.config_keys = configs
+        self.state_keys = states
+        self.action_keys = actions
+        self.sim_physics = SimulatorSession()
```

Comment:
ahhh okay, so you have both the original moab under `sim_physics` and the ddm model under `dd_model`. And you're running them both in the `test_random_policy` loop.

Reply:
Yes, both run in the loop and get logged in separate files.